remix-astwalker/src/types.ts

```diff
@@ -0,0 +1,22 @@
+export interface Node {
+  ast: any | null,
+  source: string | null
```

Comment:
try to avoid as much as possible | null if not needed. In strict mode you'll get an error if you don't first check if the value exist.